Sources/XCTest/XCTAssert.swift

```diff
@@ -267,7 +267,7 @@ public func XCTAssertNil(@autoclosure expression: () -> Any?, @autoclosure _ mes
         if value == nil {
             return .Success
         } else {
-            return .ExpectedFailure("\"\(value)\"")
+            return .ExpectedFailure("\"\(value!)\"")
```

Comment:
Could we reverse the conditional here, to avoid the `!`? Exclamation points make me nervous. :sweat_smile:

How about:

```swift
if let value = expression() {
    return .ExpectedFailure("\"\(value)\"")
} else {
    return .Success
}
```